R/uvhydrograph-render.R

```diff
@@ -417,53 +417,30 @@ createSecondaryPlot <- function(uvInfo, timeInformation, secondarySeriesList,
   return(plot_object)
 }
 
-#' Compute the y-axis real interval, based on a heuristic.
+#' Compute the y lims, y-lims will ensure all of the corrected points are shown, but not necessarily all of the uncorrected points
 #' @param corr.value.sequence A sequence of corrected time series points.
 #' @param uncorr.value.sequence A sequence of uncorrected time series points.
-#' @return The y-axis real interval, as ordered-pair vector.
-YAxisInterval <- function(corr.value.sequence, uncorr.value.sequence) {
-  return(c(
-      YOrigin(corr.value.sequence, uncorr.value.sequence),
-      YEndpoint(corr.value.sequence, uncorr.value.sequence)
-  ))
-}
-
-#' Compute the y-axis origin, based on a heuristic.
-#' @param corr.value.sequence A sequence of corrected time series points.
-#' @param uncorr.value.sequence A sequence of uncorrected time series points.
-#' @return The y-axis real interval origin.
-YOrigin <- function (corr.value.sequence, uncorr.value.sequence) {
+#' @return The y-lim
+calculateYLim <- function(corr.value.sequence, uncorr.value.sequence) {
+  bufferPercent <- .30 #percent of corrected range allowed to extend to include uncorrected points. If lims of uncorrected points within percent. 
+  
   min.corr.value <- min(corr.value.sequence, na.rm = TRUE)
   min.uncorr.value <- min(uncorr.value.sequence, na.rm = TRUE)
   
-  # if minimum corrected value is below or equal to minimum uncorrected, or if
-  # the minimum uncorrected value is less than 70% of the minimum corrected
-  # value
-  if (min.corr.value <= min.uncorr.value || min.uncorr.value < 0.70 * min.corr.value)
-    y.origin <- min.corr.value # use minimum corrected value as y-axis origin
+  if (min.corr.value <= min.uncorr.value || min.uncorr.value < (1 - bufferPercent) * min.corr.value)
+    y.bottom <- min.corr.value 
   else
-    y.origin <- min.uncorr.value # use minimum uncorrected value as y-axis origin
-
-  return(y.origin)
-}
-
-#' Compute the y-axis endpoint, based on a heuristic.
-#' @param corr.value.sequence A sequence of corrected time series points.
-#' @param uncorr.value.sequence A sequence of uncorrected time series points.
-#' @return The y-axis real interval endpoint.
-YEndpoint <- function (corr.value.sequence, uncorr.value.sequence) {
+    y.bottom <- min.uncorr.value 
+  
```

Comment:
Wanna put brackets around this just to make it explicit what is contained in the else statement?